src/main.js

```diff
@@ -13,22 +13,71 @@
  *
  */
 
-// Does ECN Viewer need port 80 ?
-const requireElevated = process.argv[2] === 'start' && (!process.env.VIEWER_PORT || process.env.VIEWER_PORT === '80')
-
+const Cli = require('./cli')
+const daemon = require('./daemon')
+const config = require('./config')
 const isElevated = require('is-elevated')
+const request = require('request-promise')
+
+const isHTTPS = () => {
+  const sslKey = config.get('Server:SslKey')
+  const devMode = config.get('Server:DevMode')
+  const sslCert = config.get('Server:SslCert')
+  const intermedKey = config.get('Server:IntermediateCert')
+  return !devMode && sslKey && sslCert && intermedKey
+}
+
+const getJSONFromURL = async (uri) => request({
+  uri,
+  json: true
+})
+
+const isDaemonElevated = async () => {
+  // If it is running and you can see it, you have enough permission to move forward
+  if (daemon.status() !== 0) {
+    return false
+  }
+  const apiPort = config.get('Server:Port')
```

Comment:
`config.get` accepts second parameter for default value.  It would be good to send `51121` here. You can do the same for the above `config.get` calls.